src/2_mockFn.test.js

```diff
@@ -1,3 +1,17 @@
-// Your tests here...
-// You'll practice the basics of jest mocks
-it("...", () => {});
+import { guardMove } from "./2_mockFn";
+
+describe("Moving a Guard", () => {
+  test("Sends a move or run action to the guard", () => {
```

Comment:
As I said in the video, you need to create 2 tests (or even 3).

```
it('Given the action "walk", it calls the moveCallback and returns its value", () => { ... })
it('Given the action "run", it calls the moveCallback and returns its value", () => { ... })
it('Given any other action, it does not call the moveCallback and returns null", () => { ... })
```

p.s `it()` means the same as `test()`